libs/shared/utils/src/lib/simpleEncryption.ts

```diff
@@ -0,0 +1,44 @@
+import { Base64 } from 'js-base64'
+import { createCipheriv, createDecipheriv, createHash } from 'crypto'
+
+const ALGORITHM = 'aes-256-cbc'
+
+/**
+ *
+ * @param text The string you wish to hide
+ * @param key You secret key
+ * @returns URL safe Base64
+ */
+export const encrypt = (text: string, key: string): string => {
```

Comment:
Maybe we should call this something other than `encrypt`? `maskString`, `maskData`?